scripts/QR Code Generator/qr_code_gen.py

```diff
@@ -0,0 +1,33 @@
+# Importing the necessary packages
+import pyqrcode
+import png
+from argparse import ArgumentParser
+import validators
+import os
+
+# Constructing an Argument Parser
+ap = ArgumentParser()
+ap.add_argument("-l", "--link", required=True, help="Path to the image folder")
```

Comment:
Change the help description I think it should be concerning the link instead of image dir.